module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraApplicationLogo.md

```diff
@@ -64,10 +67,11 @@ This example shows how to retrieve the application logo for an application that
 ## Parameters
 
 ### -FileName
+
 If provided, the application logo is copied to the file who's name is provided in this parameter.
```

Comment:
If provided, the application logo is saved to the file using the specified file name.

Reply:
Added. Thank you.